src/sentry/release_health/duplex.py

```diff
@@ -811,9 +812,10 @@ def _dispatch_call_inner(
             try:
                 with timer("releasehealth.metrics.duration", tags=tags, sample_rate=1.0):
                     metrics_result = metrics_fn(*args)
-            except InvalidParams:
-                # This is a valid result from metrics, not a crash
-                # -> no need to fall back to session_fn, just re-raise
+            except (InvalidParams, InvalidSearchQuery):
+                # This is a valid result from metrics, not a crash -> no need to fall back to session_fn, just re-raise
+                # There is no need to also send an issue to Sentry on InvalidSearchQuery exception instances as these
+                # are not crashes, and just the result of incompatible search filters.
                 raise
```

Comment:
You have a `raise` here, so it will re-throw the exception as an uncaught one. The outcome is the same as if you were directly running the code, for that case. Errors will be sent to Sentry. Maybe you want to `pass`?

Reply:
No its caught here https://github.com/getsentry/sentry/blob/4acda04fa130c6d7c776b393b9a91dfbc3f55a7e/src/sentry/api/bases/organization_events.py#L134-L143

Reply:
> A huge spike of InvalidSearchQuery seems a good way to identify cases where customers can't make a query. Do we have any plans/alternatives to identify such cases and get alerted?

I do not think that is necessary because the allowed filters are actually shown in the UI, and they are only `release` and basically semver fields, anything else will raise this exception

Reply:
LGTM then!